### DISCUSSION TITLE
Entry size too large? Getting "An error has occurred: 413"

### DISCUSSION THREAD

**Discussion by hdoshi2** [supabase/supabase#4639] [Questions] [answered]
Hello, 

I am attempting to add a json file as an entry in my supabase table. I believe if the json file is larger than 1mb the insert fails and returns "An error has occurred: 413". Is there any way around this? Would it be possible to increase that entry max size limit? 

Alternatively, I could use storage bucket to store the json file, but I would prefer to save in the table if possible (mainly because I am using legacy python 2 for an older project which does not have any supabase library support).

Thanks,
Hardik

**Comment by inian** [comment, marked as answer]
Hi @hdoshi2, we are tracking the issue here https://github.com/supabase/supabase/issues/4391

Looks like you have considered Storage already, that would be a good workaround till the issue is fixed in the dashboard
